src/components/TypeaheadTable.js

```diff
@@ -117,7 +128,9 @@ TypeaheadTable.defaultProps = {
     placeholder: '',
     table: { rows: [] },
     label: '',
-    debounce: null
+    debounce: 500,
+    minimumSearchTermLength: 1,
+    queryString: ''
```

Comment:
Should this have null as default?

Reply:
useSearch has a default arg of 500 so null works